rules/relational-database-design/rule.md

```diff
@@ -195,25 +196,38 @@ So to fix it, Book needs to become a separate table.
 
 ERDs are a way of visualizing the relationships between different entities and their cardinality. 
 
+`youtube: https://www.youtube.com/watch?v=6uwuNRUUimY`
```

Comment:
Recommend line break after the YouTube video to ensure that the Figure caption is on the next line.